### PR TITLE
add inverse solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/inverseRobot.js [hunk 1]
```diff
@@ -7,7 +7,17 @@
...
-  // write code here
+  const robotKeys = Object.keys(robot);
+  const newRobot = {};
+
+  for (let i = 0; i < robotKeys.length; i++) {
+    if (newRobot.hasOwnProperty(robot[robotKeys[i]])) {
+      return null;
+    }
+    newRobot[robot[robotKeys[i]]] = robotKeys[i];
+  }
```
Comment (choeqq) [inline src/inverseRobot.js:18]: Please use variable for `robotKeys[i]`

**Review by choeqq — CHANGES_REQUESTED** [review]

**Review by choeqq — CHANGES_REQUESTED** [review]
Waiting for any updates @WaveOfDandelions 👀

File: src/inverseRobot.js (hunk 1 above)
Comment (WaveOfDandelions) [reply src/inverseRobot.js:18]: > Please use variable for `robotKeys[i]`

Can you show me example what you exactly mean?
 I have variable before in code const robotKeys = Object.keys(robot);

File: src/inverseRobot.js [hunk 2]
```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotKeys = Object.keys(robot);
+  const newRobot = {};
+
+  for (let i = 0; i < robotKeys.length; i++) {
+    if (newRobot.hasOwnProperty(robot[robotKeys[i]])) {
```
Comment (choeqq) [inline src/inverseRobot.js:14]:
```suggestion
  for (let i = 0; i < robotKeys.length; i++) {
   const robotKey = robotKeys[i];
    if (newRobot.hasOwnProperty(robot[robotKey])) {
```

**Review by choeqq — CHANGES_REQUESTED** [review]
Is that fine?

File: src/inverseRobot.js (hunk 1 above)
Comment (asakevych) [reply src/inverseRobot.js:18]: 1. Kacper means you need to create variable with `robotKeys[i]` and use it inside loop, so your code would be easier to read
2. I think you can use `for in` loop through object insted of creating array with robot keys to simplify your solution

**Review by asakevych — CHANGES_REQUESTED** [review]

**Comment by WaveOfDandelions** [comment]
> Is that fine?

Ok I will rewrite

**Review by choeqq — APPROVED** [review]
Good job ✅

**Pull request closed without merging**